docs/diagrams/ModelClassDiagram.puml

```diff
@@ -32,25 +35,25 @@ Class Tag
 Class HiddenOutside #FFFFFF
 HiddenOutside ..> Model
 
-AddressBook .up.|> ReadOnlyAddressBook
+StoreMando .up.|> ReadOnlyStoreMando
 
 ModelManager .up.|> Model
 Model .right.> ObservableList
-ModelManager o--> "1" AddressBook
+ModelManager o--> "1" StoreMando
 ModelManager o-left-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
-AddressBook *--> "1" UniquePersonList
-UniquePersonList o--> "*" Person
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
-Person *--> "*" Tag
+StoreMando *--> "1" UniqueItemList
+UniqueItemList o--> "*" Item
+Item *--> "1"ItemName
+Item *--> "1"Quantity
+Item *--> "1"Location
+Item *-right-> "0..1"ExpiryDate
+Item *-right-> "*" Tag
 
-Name -[hidden]right-> Phone
-Phone -[hidden]right-> Address
-Address -[hidden]right-> Email
+ItemName -[hidden]right-> ExpiryDate
+ExpiryDate -[hidden]right-> Location
```

Comment:
What does this look like?

Reply:
<img width="393" alt="Screenshot 2021-03-22 at 10 14 32 PM" src="https://user-images.githubusercontent.com/66583841/112003583-fbef9780-8b5b-11eb-8ab4-f03025d79256.png">

It looks like this now, the hidden arrows wont be displayed.